src/core/store/modules/tokens/tokens.selectors.ts

```diff
@@ -106,6 +87,7 @@ export function createToken(props: CreateTokenProps): TokenView {
     description: tokenData?.metadata?.description ?? '',
     website: tokenData?.metadata?.website ?? '',
     isZapable: tokenData?.supported.zapper ?? false,
```

Comment:
Should we add other supported zaps here?
```suggestion
    isZapable: tokenData?.supported.zapper || tokenData?.supported.ftmApeZap ?? false,
```

Reply:
imo, this should be meant for the PR that implements custom zaps 😁 

Also, `supported.zapper` can start to get deprecated in favor of the explicit zapper contract, like `supported.zapperZapIn`

Reply:
Yes, let's leave it for the other PR